### ISSUE TITLE
About result in OCT

### ISSUE THREAD

**Issue by qiaoqiangPro** [Negin-Ghamsarian/Transformation-Invariant-Self-Training-MICCAI23#3]
Hello, negin.
My Test_Dice is about 30%, but your paper say 40.0%, I want to know what wrong in my practice.
![1704187023149](https://github.com/Negin-Ghamsarian/Transformation-Invariant-Self-Training-MICCAI2023/assets/80230735/6de2a4f5-f930-4d34-ad02-201715f80ba9)

**Comment by Negin-Ghamsarian** [comment]
Hello! You should get results very close to the ones reported in the paper if you use the same settings (Section 3). The hyperparameters can be found and changed in the config file. For instance, if you do not set hard_label_thr = 0.85, you can expect degraded performance compared to the reported results in Table 1.

**Comment by qiaoqiangPro** [comment]
> Hello! You should get results very close to the ones reported in the paper if you use the same settings (Section 3). The hyperparameters can be found and changed in the config file. For instance, if you do not set hard_label_thr = 0.85, you can expect degraded performance compared to the reported results in Table 1.

Thank you very much for your answer, I am implementing your data division exactly as it is in `Config_ENCORE_AugLoss_UnsupOnTrain_T85.py`, I pushed a request aimed at fixing the division of the data into different folds, if I follow the source code, it will make the second fold less than the first fold, and the third fold less than the second fold, and this will have an impact on the results of my reproduction as well, I need your help!

**Comment by Negin-Ghamsarian** [comment]
As I mentioned in response to your previous question, you should use the TrainIDs provided under TrainIDs_RETOUCH_DA in case you would like to reproduce our results.

**Comment by qiaoqiangPro** [comment]
> As I mentioned in response to your previous question, you should use the TrainIDs provided under TrainIDs_RETOUCH_DA in case you would like to reproduce our results.

Yeah, you're right.
Thank you for your help. I followed the scheme you provided to divide, there is a slight difference in the number of data in the CSV file after division, I found that the IRF file after dataset division for SpectralisVsTopcon4_1.csv has the patient TRAIN047, but there is a slice of TRAIN047_11 in my file, but not in your csv file. viewed the original image and there is mask, this may be the reason for the inconsistent number with you, do you know why this is?

![image](https://github.com/Negin-Ghamsarian/Transformation-Invariant-Self-Training-MICCAI2023/assets/80230735/ccb2d4d2-eb25-4ef4-b8b3-d5830d307f7f)

**Comment by qiaoqiangPro** [comment]
> > As I mentioned in response to your previous question, you should use the TrainIDs provided under TrainIDs_RETOUCH_DA in case you would like to reproduce our results.
> 
> Yeah, you're right. Thank you for your help. I followed the scheme you provided to divide, there is a slight difference in the number of data in the CSV file after division, I found that the IRF file after dataset division for SpectralisVsTopcon4_1.csv has the patient TRAIN047, but there is a slice of TRAIN047_11 in my file, but not in your csv file. viewed the original image and there is mask, this may be the reason for the inconsistent number with you, do you know why this is?
> 
> ![image](https://private-user-images.githubusercontent.com/80230735/293814690-ccb2d4d2-eb25-4ef4-b8b3-d5830d307f7f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQyNDMzNzIsIm5iZiI6MTcwNDI0MzA3MiwicGF0aCI6Ii84MDIzMDczNS8yOTM4MTQ2OTAtY2NiMmQ0ZDItZWIyNS00ZWY0LWI4YjMtZDU4MzBkMzA3ZjdmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTAzVDAwNTExMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ3N2I0ZDYxZmI4MzgwOGNlYTZlOTdlMmVhNzIyZjE3YjhiNWIxNDcyNDQ5OTM4MjY3NTYzNDMyODQ2NzgwOTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.S6q0B7aC62J9jNkDjFrCeSPEn_w_wbb5ysvnwzKnOy4)

Oh, make them in one image  as follow:
![image](https://github.com/Negin-Ghamsarian/Transformation-Invariant-Self-Training-MICCAI2023/assets/80230735/42cd703e-745c-4a64-b2db-2fff2d8b3a1d)

**Comment by Negin-Ghamsarian** [comment]
> TRAIN047_11

This case exists in SpectralisVsTopcon4_1.csv at line 246.

**Comment by qiaoqiangPro** [comment]
> > TRAIN047_11
> 
> This case exists in SpectralisVsTopcon4_1.csv at line 246.

Oh sorry,This was an oversight on my part, I used the data division in your `TrainIDs_RETOUCH_DA` folder and did the training again, the results were still unsatisfactory, in which I checked the `Config_ENCORE_AugLoss_ under the folder `configs_RETOUCH_DA_scSENet_ST4`. UnsupOnTrain_T85.py` code, I simply set image_transforms = Compose([RandomApply_Customized([
                                   ColorJitter(brightness=0.7, contrast=0.7, saturation=0.7, hue=0)...

Instead of ColorJitter(brightness=0.6, contrast=0.6, saturation=0.4, hue=0) in your example file, I will recover your settings and try again, currently the dice for quadruple fold is 29%... I think there is a lot of difference and need your help.
![image](https://github.com/Negin-Ghamsarian/Transformation-Invariant-Self-Training-MICCAI2023/assets/80230735/beef79c7-e6e3-4c17-a0f3-bc4f39d928f1)
